docs/source/changelog.rst

```diff
@@ -0,0 +1,136 @@
+Changelog
+=========
+
+0.3.0.dev0 (continuous)
+#######################
+
+.. Commented out
+   Features
+   --------
+
+.. Commented out
+   Bug fixes
+   ---------
+
+Documentation
+-------------
+
+* Instructions for profiling slow tests (:issue:`447`, :pr:`448`)
+* Include review of issues, PRs and release notes in release checklist (:pr:`445`)
+* Change management (:issue:`443`, :pr:`451`)
+
+.. Commented out
+   Obsolescence
+   ------------
+
+Miscellaneous
+-------------
+
+* Performance improvements for unit tests (:issue:`447`, :pr:`448`)
+
+0.2.2
+#####
+
+Point release to fix a number of minor issues, most notably PR :pr:`439` that
+should have been merged for version 0.2.
+
+Bug fixes
+---------
+
+* Trigger a timeout when guessing parameters for HDF5 takes too long (:issue:`440` , :pr:`449`)
+* Improved error and validation handling when opening files with GUI (:issue:`433,442`)
```

Comment:
This is actually not part of the point release, as at least the GUI related changes are a bit entangled with TypeScript updates etc., added a new HTTP endpoint etc.

What do you think about just backporting [this commit](https://github.com/LiberTEM/LiberTEM/commit/ec74c1346d93eff58d9e2201a7ead5af7aa7cf44) and changing the log entry to something like "Slightly improved error [...]"?

Reply:
Ok! I've updated the file accordingly.